src/test/java/com/wavesplatform/wavesj/json/deser/AssetDetailsDeserTest.java

```diff
@@ -0,0 +1,28 @@
+package com.wavesplatform.wavesj.json.deser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.wavesplatform.wavesj.AssetDetails;
+import com.wavesplatform.wavesj.json.WavesJsonMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class AssetDetailsDeserTest{
```

Comment:
I wonder why **TransactionDeserTest** was not used as a parent here and you did a copy/past ?
Do you have some issues with this base class ?

Reply:
I found it to be semantically confusing if I had used TransactionDeserTest as a parent because we're not dealing with a class which extends Transactions. That's all.

Reply:
Agree